aws/data_source_aws_organizations_delegated_administrators_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func TestAccDataSourceAwsOrganizationsDelegatedAdministrators_basic(t *testing.T) {
+	var providers []*schema.Provider
+	dataSourceName := "data.aws_organizations_delegated_administrators.test"
+	servicePrincipal := "config-multiaccountsetup.amazonaws.com"
+	dataSourceIdentity := "data.aws_caller_identity.delegated"
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Since this test updates settings at the account level, it should not be run in parallel with other tests

```suggestion
	resource.Test(t, resource.TestCase{
```